src/battle.rs

```diff
@@ -505,6 +522,52 @@ pub (crate) fn attack_button_handler (
     }
 }
 
+fn calculate_damage(player: &MonsterBundle, player_action: usize, 
```

Comment:
Again a naming nitpick, but I dislike calling the player's monster a "player" when it's not. I expected that a player's monster might drop to 0 HP without the player losing instantly, as they have other monsters. If this isn't the case, we have to make that more clear to players.

Reply:
Very good point. Open a issue so we can resolve this when we introduce monster bags.